packages/framework/src/commands/Command.ts

```diff
@@ -0,0 +1,25 @@
+import type {
+  APIApplicationCommandOption,
+  RESTPostAPIChatInputApplicationCommandsJSONBody,
+} from "discord-api-types/v10";
+import type { ChatInputInteraction } from "../structs";
+
+export interface DisploySlash {
+  name: string;
+  description: string;
+  options?: APIApplicationCommandOption[];
+}
+
+export class Command {
+  public constructor(public options: DisploySlash) {}
+
+  public slashRun?(interaction: ChatInputInteraction): void;
+
+  public toJson(): RESTPostAPIChatInputApplicationCommandsJSONBody {
+    return {
+      name: this.options.name,
+      description: this.options.description,
+      options: this.options.options,
+    };
```

Comment:
```suggestion
    return {
      ...this.options,
    };
```

Reply:
resolved in https://github.com/Disploy/disploy/pull/19#:~:text=%E2%80%A6-,fb99aa3,-Merge%20state